distribution/pom.xml

```diff
@@ -256,6 +256,13 @@
                                     <overWrite>true</overWrite>
                                     <outputDirectory>target/ext/broker</outputDirectory>
                                 </artifactItem>
+                                <artifactItem>
+                                    <groupId>org.wso2.securevault</groupId>
+                                    <artifactId>org.wso2.securevault</artifactId>
+                                    <version>1.0.0-wso2v2</version>
```

Comment:
Shall we extract the version to a property

Reply:
:+1:  done